### ISSUE TITLE
NTR: subgenual anterior cingulate cortex

### ISSUE THREAD

**Issue by Jchaffer787** [obophenotype/uberon#3005]
**Preferred term label:** subgenual anterior cingulate cortex

**Synonyms** N/A

**Definition (free text, please give PubMed ID)**
The anterior cingulate cortex (ACC) is divided into pregenual (pACC) and subgenual (sACC) regions. Compared with the subgenual portion of ACC (sACC), the pregenual portion (pACC) has more widespread connections with the lateral prefrontal cortex and much fewer with the amygdala. It also has fewer projections to the ventral striatum.
(https://neuro.psychiatryonline.org/doi/10.1176/jnp.23.2.jnp121)

**Parent term (use https://www.ebi.ac.uk/ols/ontologies/uberon)**
anterior cingulate cortex

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0003-1653-8449
